target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8064-db149.dts

```diff
@@ -14,8 +14,7 @@
 		};
 	};
 
-	alias {
-		serial0 = &gsbi2_serial;
+	aliases {
 		mdio-gpio0 = &mdio0;
```

Comment:
If we are going to drop serial0, then IMO we can also drop mdio-gpio0, which is for [mdio-gpio](https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/net/mdio-gpio.txt#L17).